docs/blueprint-designer-guide/blueprint-quickstart-guide.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_position: 3
+title: Blueprint Quickstart Guide
+---
+
+Now that you have seen how Torque works, it’s time to link your assets to Torque and see how easy what you can do with them.
+
+## Prerequisites
+
+* Space
```

Comment:
Rephrase to "you have created your own space" or something. Otherwise it's not clear, Sample is also a space.